### PR TITLE
Gf2016 fix

### PR DESCRIPTION
**Pull request by kalapi**
@alexeiva Hey, i've fixed the merge conflicts. Please check and merge into your branch. Then I can merge it into master.

### CODE REVIEW

**Comment by kalapi** [comment]
@googlebot I confirm that these commits were authored by me. The email registered with Google is the same as the one configured in my Git config.

File: tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/PLUS_drawn_215.txt [hunk 1]
```diff
@@ -85,51 +85,51 @@
 0x2122  ™ trademark
 0x005E  ^ asciicircum
 0x007E  ~ asciitilde
-0x0300  ̀  gravecomb
+0x0300  ̀ gravecomb
```
Comment (davelab6) [inline tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/PLUS_drawn_215.txt]: I'm not sure about the changes to PLUS_drawn_215.txt

Comment (kalapi) [inline tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/PLUS_drawn_215.txt]: Why? the only thing I did was remove an extra space character before the glyph names. Which is a legacy from the original .nam files.

Comment (davelab6) [inline tools/encodings/GF 2016 Glyph Sets/glyphTypeSorting/PLUS_drawn_215.txt]: Ah okay, the github diff preview looked like the space was.... eliminated

**Pull request merged**